.github/workflows/sync-valint-version.yaml

```diff
@@ -0,0 +1,44 @@
+name: Sync Valint Version
+
+on:
+  pull_request:
+    types:
+      - closed
+
+permissions:
+  contents: write
+
+jobs:
+  sync_version:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master')
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Read version from file
+        id: read_version
+        run: echo "version=$(cat .version)" >> $GITHUB_OUTPUT
+
+      - name: Check if tag exists
+        id: check_tag
+        run: |
+          tag_name=$(echo "${{ steps.read_version.outputs.version }}")
+          git fetch --tags
+          if git rev-parse "$tag_name" >/dev/null 2>&1; then
+            echo "Tag $tag_name already exists"
+            echo "::set-output name=tag_exists::true"
+          else
+            echo "Tag $tag_name does not exist"
+            echo "::set-output name=tag_exists::false"
```

Comment:
this is no longer how github support set output. 
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-masking-a-generated-output-within-a-single-job

Reply:
fixed in 4e810d184318d284c86304025bd1e74334316680